portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -240,12 +240,37 @@ definition {
 	}
 
 	@description = "Verify that the Field Title can be edited"
-	@ignore = "true"
 	@priority = "5"
 	test TitleCanBeEdited {
 		property portal.acceptance = "false";
```

Comment:
Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.

Reply:
Done.
Thanks.